src/commands/bundle.js

```diff
@@ -214,7 +214,11 @@ let command = ({
       name: 'progress',
       description:
         'Display bundle compilation progress with different verbosity levels',
-      default: 'compact',
+      default: () => {
+        // Ensure that we don't trip Xcode's error detection. 'verbose' is the
+        // only level that doesn't make Xcode think that the bundle failed.
+        return !process.stdin.isTTY ? 'verbose' : 'compact';
```

Comment:
I propose defaulting to `expanded` instead of `verbose`. On my set up, `expanded` and `verbose` had the exact same amount of lines (2846), but I feel like expanded conveys the opposite of `compact`.

Also, not sure if the comments are necessary – If someone reviews the PR and commit message that should be sufficient for understanding the reasons behind the change.

Reply:
I tried `expanded` first, and it caused issues when run in Azure DevOps. The comment is clearer as lines may be moved by later commits and will be harder to track down.